### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by lgp3212**
tests for metadata and preprocessing

### CODE REVIEW

File: script/preprocessing.py [hunk 1]
```diff
@@ -135,6 +135,8 @@ def musdb_track_info(data_home, tid):
 
     track_metadata["sound_class"] = sound_class
 
+    print("TRACK METADATA: ", track_metadata)
+
```
Comment (giovana-morais) [inline script/preprocessing.py:139]:
```suggestion
```

File: script/mix.py [hunk 2]
```diff
@@ -1,306 +1,513 @@
...
 import random
 import uuid
-import soundfile as sf
+
 import librosa
-import json
 import numpy as np
+import soundfile as sf
 
+#import timeit
```
Comment (giovana-morais) [inline script/mix.py:12]:
```suggestion
```

**Comment by giovana-morais** [comment]
@lgp3212 there are a lot of conflicts in the mix.py file. please resolve them and then we can proceed with the review!

File: tests/test_md.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import pytest
+import os
+
+from metadata import get_sound_class
+
+# NOTE: currently running this out of stem_mixer/script, might need to adjust paths later on
+
+@pytest.mark.parametrize(
```
Comment (giovana-morais) [inline tests/test_md.py:8]: maybe we add an example that does not belong to musdb/brid but still has a clear sound class?

**Review by giovana-morais — APPROVED** [review]

**Review by giovana-morais — CHANGES_REQUESTED** [review]
i approved by mistake :sweat_smile: we need to fix conflicts and then add tests for the mix module!

File: script/mix.py [hunk 4]
```diff
@@ -193,19 +196,17 @@ def select_tracks(data_home, n_stems, n_harmonic, n_percussive, tempo_bin_harmon
...
 
     except ValueError: # if any error gets thrown, invalid mixture
         invalid_mixture = True
-        #print("invalid, unexpected error")
+        print("invalid, unexpected error")
 
-    print("selected stems:")
-    for stem_name in selected_stems.keys():
-        print(stem_name)
+    print("selected stems: ", selected_stems)
```
Comment (giovana-morais) [inline script/mix.py:209]:
```suggestion
```

File: tests/test_md.py (hunk 3 above)
Comment (lgp3212) [reply tests/test_md.py:8]: ok i can add this

**Comment by giovana-morais** [comment]
closing this because i merged it by mistake by merging the documentation branch.

**Pull request closed without merging**